coalib/coala_main.py

```diff
@@ -60,6 +62,10 @@ def run_coala(console_printer=None,
     :param arg_parser:              Instance of ArgParser that is used to parse
                                     non-setting arguments.
     :param arg_list:                The CLI argument list.
+    :param args:                    The parsed CLI arguments.
```

Comment:
what is the args behaviour if this isn't given (none)? IMO docs should cover that